omod/src/main/webapp/resources/dhisconnector.js

```diff
@@ -316,7 +316,7 @@ function saveMapping(event) {
 			contentType : "application/json; charset=utf-8",
 			dataType : "json",
 			success : function(data) {
-				window.location = '../../module/dhisconnector/runReports.form';
+				window.location = '../../module/dhisconnector/manageMappings.form';
```

Comment:
thanks @Piumal1999  do we need this change?

Reply:
Yeah @HerbertYiga. After a user creates a mapping it will redirect to the manage mappings UI now. Create Mapping and Manage Mapping pages require the same privilege. 
Previously it was redirecting to the run reports Page which required a different privilege.